code/modules/clothing/head/helmet.dm

```diff
@@ -17,6 +17,9 @@
 		"Drask" = 'icons/mob/species/drask/helmet.dmi'
 		)
 
+/obj/item/clothing/head/helmet/modern

```

Comment:
65536, but I believe types were one of the limited things in DM that were uncapped to "functionally infinite"